src/components/media-loader.js

```diff
@@ -111,9 +115,9 @@ AFRAME.registerComponent("media-loader", {
       this.loadingClip.play();
     }
     this.el.setObject3D("mesh", mesh);
-    this.hasBakedShapes = !!(this.el.body && this.el.body.shapes.length > 0);
-    this.setShapeAndScale(true);
+    this.setScale(true);
     delete this.showLoaderTimeout;
+    this.addShape("box", "loader");
```

Comment:
May be nice to export constants AMMO_BOX, AMMO_HULL, etc to help indicate that these are specific strings that are meaningful to ammo components